database/sqlite/sqlite_metadata.c

```diff
@@ -1281,6 +1291,27 @@ void metaqueue_delete_dimension_uuid(uuid_t *uuid)
     queue_metadata_cmd(METADATA_DEL_DIMENSION, use_uuid, NULL);
 }
 
+void metaqueue_store_claim_id(uuid_t *host_uuid, uuid_t *claim_uuid)
+{
+    if (unlikely(!host_uuid))
+        return;
+
+    uuid_t *local_host_uuid = mallocz(sizeof(*host_uuid));
```

Comment:
I am reporting here an event that has relationship not only with this PR, but considering that is also happening here, I will  also share, please, take a look in the following report:

```sh
==28173== 152 bytes in 1 blocks are still reachable in loss record 3,278 of 4,108
==28173==    at 0x48475FF: calloc (vg_replace_malloc.c:1328)
==28173==    by 0x44A0A4: malloc_trace_find_or_create (libnetdata.c:219)
==28173==    by 0x44A4D5: mallocz_int (libnetdata.c:253)
==28173==    by 0x58E8B1: metaqueue_store_claim_id (sqlite_metadata.c:1299)
==28173==    by 0x66D043: load_claiming_state (claim.c:173)
==28173==    by 0x42B869: main (main.c:1957)
```

as you can see `queue_metdata_cmd` is not cleaning the allocated memory (ping @underhood ), probably we need to address this. @MrZammler do you want try a fix at least for this reported leak?

Reply:
Hi Thiago, thanks, yes, will check.

Reply:
After a check with @MrZammler we discovered that issue is appearing only when we use `NETDATA_TRACE_ALLOCATIONS` (ping @stelfrag ), considering that I do not expect users using this flag, I am going to approve the PR.